setup.py

```diff
@@ -123,5 +123,7 @@
                  ['config/liota.conf', 'config/logging.json']),
                 (os.path.abspath(os.sep) + '/../etc/liota',
                  ['BSD_LICENSE.txt', 'BSD_NOTICE.txt']),
+                (os.path.abspath(os.sep) + '/../etc/init.d',
```

Comment:
Don`t copy forcefully in /etc/init.d, make it a manual step and document in README.md if user wants to use this functionality.